src/test/resources/com/cognifide/gradle/aem/test/debug/additional/debug.json

```diff
@@ -229,6 +229,8 @@
 
       "deployDistributed": false,
       "uploadForce": true,
+      "downloadRetryTimes": 3,
```

Comment:
why retry for download is required? please remove it or align to changes applied within 5.0.1 (use `AemConfig.Retry`)